lib/TraackrApi/Posts.php

```diff
@@ -9,8 +9,7 @@ public static function lookup($p = array(
       'lang' => 'all',
       'include_entities' => false,
       'count' => 25,
-      'page' => 0,
-      'force_vit_legacy' => true) ) {
```

Comment:
Removed this from the default, because the default value should be false, which convertBool() will handle. I decided to make the default false after playing around with the two FFs (enable vit 2 , disable vit 2) ... it was excessive.